tests/ci/docker_server.py

```diff
@@ -355,13 +373,13 @@ def main():
             repo_urls[arch] = f"{args.bucket_prefix}/{build_name}"
         if args.allow_build_reuse:
             # read s3 urls from pre-downloaded build reports
-            if "clickhouse-server" in args.image_repo:
+            if "clickhouse-server" in image_repo:
```

Comment:
we control now `image_repo`, and we can code it into some class even. With PACKAGES together

Reply:
I'll leave creating a class for later